src/ts/ssh/sshSession.ts

```diff
@@ -69,6 +70,12 @@ interface RequestHandler {
 	isCancelled?: boolean;
 }
 
+/**
+ * Allows SSH Server sessions to keep track of the session number for progress
+ * reporting purposes.
+ */
+let serverSessionCounter = 0;
```

Comment:
Why limit it to only server sessions? Can be used for both.

Reply:
Does it make sense in the client context?

Reply:
Thanks! As discussed, could be used for reconnections.